src/utils/config.ts

```diff
@@ -592,7 +597,7 @@ export async function getAccessToken(target: Target, checkIfExpiring = true) {
 
     let authConfig
 
-    if (refreshToken) {
+    if (refreshToken && !isRefreshTokenExpired(refreshToken)) {
```

Comment:
There's no need for the first part of this check (existence of `refreshToken`) as this is already done within the `isRefreshTokenExpired` function.

Reply:
Checking presence of `refreshToken` is for `refreshTokens()` function called from inside of the if block.

Reply:
I have toggled if else blocks and removed initial part.